Makefile

```diff
@@ -35,8 +35,9 @@ build:
 
 .PHONY: test
 test: fmt envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v -race -coverprofile=coverage.out -covermode=atomic $(shell go list ./... | grep -v -E '/cmd|/mockdevice')
-
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v -race -coverprofile=coverage.out -covermode=atomic $(shell go list ./... | grep -v -E '/cmd|/mockdevice|/pkg/telemetryservice')
+	go test -v -coverprofile=coverage_tdengine.out -covermode=atomic ./pkg/telemetryservice
```

Comment:
add `coverage_tdengine.out` to https://github.com/Edgenesis/shifu/blob/main/.github/workflows/codecov.yml#L26